### ISSUE TITLE
Application completely froze me out of using Google Chrome!!

### ISSUE THREAD

**Issue by davidmponder** [k-ivan/visual-bookmarks-chrome#122]
From the very brief positive;
the app looks to have potential.

From the absolutely (sorry) negative:
it completely locked up my google chrome account to the point where it didn't work.
the solution was that i had to go into the user data folder and delete everything related to the Visual Bookmarks (VIS BKM) extension.
problem was when it initially froze, I had to close/restart Chrome, then a msg from VIS BKM kept saying it would work in the background after restarting.
Problem was that when i restarted i had 20+ pages open and it kept trying to auto re-load, even if I clicked not to.
After deleting the extension folder i eventually got the message back saying "Keep It" (meaning VIS BKM) or "Change Back" then i chose change back and was so lucky to recover it.
I had to try doing this several times before it worked.

So in conclusion; it looks like it has potential but for me I will not be trying this ever again.

**Comment by davidmponder** [comment]
Sorry for the confusing msg, I'll try and reply again after to work to clarify in a simpler step by step breakdown of what happened. Thanks for the reply.

And I agree maybe I  was just unlucky and there may have been something strange going on with my pc at the time to cause this issue.

**Comment by k-ivan** [comment]
If it is possible to describe in more detail step by step indicating the browser version, extension version, operating system, and actions, it would be ideal.  I always try to help or sort out the problem if it possible.

**Comment by k-ivan** [comment]
I'm closing because I didn't get any details.
